app/uk/gov/hmrc/emailgateway/models/Error.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.emailgateway.models
+
+import play.api.libs.json.{Json, Writes}
+import uk.gov.hmrc.emailgateway.config.Constants
+
+sealed class Error(val code: String, val desc: String)
+
+case object MissingCorrelationId extends Error("MISSING_CORRELATION_ID", s"${Constants.xCorrelationId} header is missing from the request")
```

Comment:
I think we should add the `DownstreamConnector` errors into here; so that we have one consistent format and place for Errors to be defined; with consistent JSON writes out to the consumer.

I will update `phone-number-gateway` to do this too.